tests/playwright/sanity/container.test.js

```diff
@@ -340,7 +340,7 @@ test.describe( 'Container tests', () => {
 		await editor.useDefaultTemplate();
 	} );
 
-	test( 'Fallback image is not on top of background video', async ( { page }, testInfo ) => {
+	test( 'Fallback image is not on top of background video AND border radius with background image', async ( { page }, testInfo ) => {
```

Comment:
@hein-obox What is this snapshot for? `tests/playwright/sanity/accordion.test.js-snapshots/accordion-darwin.jpeg`